README.rst

```diff
@@ -31,6 +31,10 @@ OpenTURNS Presentations
   - `Release highlights <https://github.com/openturns/openturns.github.io/blob/master/presentation/master/userday2020relhi.pdf>`_
   - `Probabilistic analyses of a spatial vehicle equipment bay <https://github.com/openturns/presentation/blob/master/userday2020/JU_OT_2020_CNES-Study.pdf>`_
   - `Uncertainties in civil engineering: A FE-model based process for research and applications <https://github.com/openturns/presentation/blob/master/userday2020/kobe_OT_user_day_2020_v01.pdf>`_
+  
+- User day 2019
+
+  - `Overview of recently developed features in Persalys <https://github.com/openturns/presentation/blob/master/userday2019/journee_OT_2019_06_07_Phimeca.pdf>`_
```

Comment:
seems you uploaded a pptx, which github does not read, maybe it could be converted to pdf ?